ert3/config/_experiment_config.py

```diff
@@ -22,15 +23,19 @@ class Config:
 class ExperimentConfig(_ExperimentConfig):
     type: Literal["evaluation", "sensitivity"]
     algorithm: Optional[Literal["one-at-a-time"]]
+    tail: Optional[float] = None
```

Comment:
Same here. It is better to require it to be a float and default it to `0.99`

Reply:
I think there should be no default value. With a default value the evaluation experiment will also have a tail that is `0.99`, which does not make sense. So just `tail: Optional[float]`.

Reply:
You are right :)